.github/workflows/cypresstests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       with:
         fetch-depth: 0
     - name: Set up Python 3.9
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v4
```

Comment:
Hi @jtwaleson! I am not sure if this is an intentional change. Do you use new features in this PR that only exist in v4? If not, can we do this in a separate PR for traceability?

Reply:
Rebased and will create a separate PR with some GH Actions cleanup at some point.